packages/qdrant-loader/src/qdrant_loader/webhooks/handlers.py

```
@@ -147,17 +148,19 @@ async def enqueue_ingest_request(
     force: bool = False,
 ) -> dict[str, Any]:
-    """Enqueue a direct ingestion job (replaces CLI `qdrant-loader ingest`).
+    """Enqueue a BULK_INGEST job processed by the serve worker pool.
 
-    Always uses FULL_SCAN; the worker runs the same pipeline as the ingest command.
+    Routes through IngestionJobHandler so the job reuses the already-warmed
+    PipelineOrchestrator rather than spawning a standalone AsyncIngestionPipeline.
+    source_type / source / project_id may all be None to mean "ingest everything".
     """
     if source is not None and source_type is None:
         raise ValueError("source_type must be provided when source is specified.")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)